mmv1/third_party/terraform/services/compute/resource_compute_global_forwarding_rule_test.go.tmpl

```diff
@@ -473,7 +501,56 @@ resource "google_compute_url_map" "urlmap" {
 }
 `, fr, proxy, backend, hc, urlmap)
 }
-{{- end }}
+
+func testAccComputeGlobalForwardingRule_pscLabels(fr, address string) string {
```

Comment:
can we separate this into two separate tests? one for `vpc-sc` and the other for `all-apis`

Reply:
Sure, but maybe it's better to have a separate `serviceAttachement` test case?